crates/modelardb_server/src/storage/uncompressed_data_buffer.rs

```diff
@@ -250,14 +249,14 @@ pub(super) struct UncompressedOnDiskDataBuffer {
 impl UncompressedOnDiskDataBuffer {
     /// Spill the in-memory `data_points` from the time series with `tag_hash` to an Apache Parquet
     /// file in `local_data_folder`. If the Apache Parquet file is written successfully, return an
-    /// [`UncompressedOnDiskDataBuffer`], otherwise return [`IOError`].
+    /// [`UncompressedOnDiskDataBuffer`], otherwise return [`crate::error::ModelarDbServerError`].
```

Comment:
Maybe "[\`ModelarDbServerError\`]\(crate::error::ModelarDbServerError\)" instead. If it is changed here, also change it elsewhere in the file.

Reply:
Fixed.